ClientServerChat/ClientServerChat/Chat.cs

```diff
@@ -0,0 +1,121 @@
+namespace ClientServerChat;
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+/// <summary>
+/// Class chat for client server interaction.
+/// </summary>
+public class Chat
+{
+    /// <summary>
+    /// Method for choosing client or server
+    /// </summary>
+    /// <param name="command"></param>
+    public static async Task Main(string[] command)
+    {
+        if (command.Length == 1)
+        {
+            int port = int.Parse(command[0]);
+            Console.WriteLine("Using:");
+            Console.WriteLine("  Server: ChatApp <port>");
+            await RunServerAsync(port);
+        }
+        else if (command.Length == 2)
+        {
+            string ipAddress = command[0];
+            int port = int.Parse(command[1]);
+           
+            Console.WriteLine("Using:");
+            Console.WriteLine("  Client: ChatApp <ipAddress> <port>");
+            await RunClientAsync(ipAddress, port);
+        }
+    }
+    /// <summary>
```

Comment:
```suggestion
    }
    
    /// <summary>
```
Тут и ниже. Вообще, между методами всегда ставится пустая строчка.